applications/fluid_structure_interaction/perpendicular_flap/application.h

```diff
@@ -336,7 +336,7 @@ class Application : public FluidFSI::ApplicationBase<dim, Number>
 
     for(auto cell : this->grid->triangulation->active_cell_iterators())
     {
-      for(unsigned int f = 0; f < dealii::GeometryInfo<dim>::faces_per_cell; ++f)
+      for(auto const & f : cell.face_indices())
```

Comment:
```suggestion
      for(auto const & f : cell->face_indices())
```

Reply:
@DavidSCN It seems as if the perpendicular_flap example is not compiled if one does not use precice. However, this test case could also be used for the ExaDG native FSI implementation. Hence, could you change/shift the "precice"-guards from the CMakeLists.txt file for this application to the precice-based FSI solver implementation?

To avoid problems as the one encountered here, we should give it a try to only guard the precice instance in the precice-based FSI driver, but always compile the rest of the driver. I am undecided whether we should guard the whole driver (assume for example that a class in ExaDG is just renamed, so that all places using this class need to be adapted.).

Reply:
>  It seems as if the perpendicular_flap example is not compiled if one does not use precice.

That's the case, yes.

I never tried to run the case using the native ExaDG coupling. We can put into the TODO list.